less/global.less

```diff
@@ -9,12 +9,12 @@ html {
 
 html, body {
   font-family: 'Ubuntu', sans-serif;
-  color: @main-font-color;
+  color: #5E6164;
 }
 
 h1, h2, h3, h4, h5 {
   font-family: 'Russo One', sans-serif;
-  color: @header-font-color;
+  color: #23293B;
```

Comment:
Why would you switch from using the set Variable here to a hex code for color? You should be using the set variable here.